easybuild/easyconfigs/h/HDF/HDF-4.2.14-GCCcore-8.3.0.eb

```
@@ -31,14 +31,21 @@ dependencies = [
 ]
 
 preconfigopts = "LIBS='-ltirpc' "
-configopts = '--with-szlib=$EBROOTSZIP '
-configopts += 'CFLAGS="$CFLAGS -I$EBROOTLIBTIRPC/include/tirpc" '
-configopts += '--includedir=%(installdir)s/include/%(namelower)s '
+configopts = [
+    '--with-szlib=$EBROOTSZIP CFLAGS="$CFLAGS -I$EBROOTLIBTIRPC/include/tirpc" \
+    --includedir=%(installdir)s/include/%(namelower)s ',
+    '--with-szlib=$EBROOTSZIP CFLAGS="$CFLAGS -I$EBROOTLIBTIRPC/include/tirpc" \
+    --includedir=%(installdir)s/include/%(namelower)s \
+    --enable-shared --disable-fortran --disable-netcdf',  # Cannot build shared libraries and Fortran...
+    # https://trac.osgeo.org/gdal/wiki/HDF#IncompatibilitywithNetCDFLibraries
+    # netcdf must be disabled to allow HDF to be used by GDAL
+]
```

you can avoid the duplicated lines like this:

```suggestion
_configopts = '--with-szlib=$EBROOTSZIP CFLAGS="$CFLAGS -I$EBROOTLIBTIRPC/include/tirpc" '
_configopts += '--includedir=%(installdir)s/include/%(namelower)s '
configopts = [
    _configopts,
    # Cannot build shared libraries and Fortran:
    # https://trac.osgeo.org/gdal/wiki/HDF#IncompatibilitywithNetCDFLibraries
    # netcdf must be disabled to allow HDF to be used by GDAL
    _configopts + '--enable-shared --disable-fortran --disable-netcdf',
]
```

---

I've seen `local_*` as the prefix for things that shouldn't be interpreted by EasyBuild in the past, does `_*` have a similar treatment and if so, which should be preferred when?

---

Both `local_*` and `_*` are valid for local variables, see https://docs.easybuild.io/en/latest/Easyconfig-files-local-variables.html#recommended-naming-scheme-for-local-variables-in-easyconfig-files